x/concentrated-liquidity/spread_rewards_test.go

```diff
@@ -1359,6 +1360,27 @@ func (s *KeeperTestSuite) TestUpdatePosValueToInitValuePlusGrowthOutside() {
 	}
 }
 
+func NewRandomizedPositions() Positions {
```

Comment:
So the function is made but why is it unused?

Reply:
I can change the concrete `Positions` struct initializations to use this function, the issue description only mentioned creating this function, hence, I left tests' code unchanged

Reply:
Can we please either make use of this or close this? I don't see the need for introducing unused code. The author probably implied making use of this

Reply:
makes sense